google/cloud/aiplatform/metadata/artifact.py

```diff
@@ -0,0 +1,285 @@
+    ) -> "_Artifact":
+        f"""Creates a new Artifact resource.
+
+        Args:
+            artifact_id (str):
+                Required. The {artifact_id} portion of the resource name with
+                the format:
+                projects/{project}/locations/{location}/metadataStores/{metadata_store_id}/artifacts/{artifact_id}.
+            schema_title (str):
+                Required. schema_title identifies the schema title used by the artifact.
+            display_name (str):
+                Optional. The user-defined name of the artifact.
+            schema_version (str):
+                Optional. schema_version specifies the version used by the artifact.
+                If not set, defaults to use the latest version.
+            description (str):
+                Optional. Describes the purpose and content of the artifact resource to be created.
+            metadata (Dict):
+                Optional. metadata contains the metadata information that will be stored in the artifact resource.
+            metadata_store_id (str):
+                The {metadata_store_id} portion of the resource name with
+                the format:
+                projects/{project}/locations/{location}/metadataStores/{metadata_store_id}/artifacts/{artifact_id}
+                If not provided, the MetadataStore's ID will be set to "default".
+            project (str):
+                Project to create this artifact into. Overrides project set in
+                aiplatform.init.
+            location (str):
+                Location to create this artifact into. Overrides location set in
+                aiplatform.init.
+            credentials (auth_credentials.Credentials):
+                Custom credentials to use to create this artifact. Overrides
+                credentials set in aiplatform.init.
+
+        Returns:
+            artifact (_Artifact):
+                Instantiated representation of the managed Metadata Artifact resource.
+
+        """
+
+        gapic_artifact = gca_artifact.Artifact(
+            schema_title=schema_title,
+            schema_version=schema_version,
+            display_name=display_name,
+            description=description,
+            metadata=metadata,
+        )
+
+        resource_name = super().create(
+            resource_id=artifact_id,
+            resource_noun=cls._resource_noun,
+            gapic_resource=gapic_artifact,
+            metadata_store_id=metadata_store_id,
+            project=project,
+            location=location,
+            credentials=credentials,
+        )
+
+        return cls(
+            artifact_name=resource_name,
+            metadata_store_id=metadata_store_id,
+            project=project,
+            location=location,
+            credentials=credentials,
+        )
+
+    @classmethod
+    def create_resource(
+        cls,
+        client: utils.AiPlatformServiceClientWithOverride,
```

Comment:
This should be specified to the `utils.MetadataClientWithOverride`

Reply:
Done, here and other places, thanks!